server/src/events.mjs

```diff
@@ -243,6 +243,11 @@ function coreSendEvent(isBroadcast, ws, userId, method, result, msg, fSuccess, f
 
       const finalResult = ( postResult ? postResult : result );
 
+      // Error to be logged in "novalidate mode"
+      if( ! config.validate.includes("events") ) {
```

Comment:
This code seems wrong. The if part looks almost right (I think you don't want the bang), but it shouldn't automatically call fErr... it should actually validate finalResult and then only call fErr if that result didn't validate, then call fErr. Right?

Reply:
if( config.validate.includes("events") && ! finalResult ) {
        fErr.call(null, method);
      }
Do you mean this?

Error should not be logged even though if "finalResult" is undefined?

basically if user pass in "--novalidate events", config.validate value would be ["method", "response", "params"].
"events" would be removed from default array.

Reply:
I agree with Mike on this. The lack of the "event" validation (AKA "! config.validate.includes("events")") should invalidate/remove existing validation functionality. We're merely printing an error here.

Why would an error be printed at this point in the code?

Reply:
ok so what would be the validation thats needs to be added against finalResult? @kschrief  @michaelcfine